engine/src/main/java/org/terasology/engine/rendering/nui/editor/utils/NUIEditorMenuTreeBuilder.java

```diff
@@ -259,7 +259,7 @@ private void populateContextMenu(JsonTree node, MenuTree addTree, boolean isSkin
                 }
             }
         } else {
-            logger.warn("Could not get class for node {}", node.getValue());
+            logger.atWarn().addArgument(() -> node.getValue()).log("Could not get class for node {}");
```

Comment:
Only evaluated in unexpected cases, so performance impact neglectable.
```suggestion
            logger.warn("Could not get class for node {}", node.getValue()); //NOPMD
```